src/test/java/com/spotify/docker/Polling.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2014 Spotify AB.
```

Comment:
The linter might check this against the java.header file which is 2014. I can do this in a later commit if changing to 2015 breaks the build